### PR TITLE
 Add short description about folder structure with few minor changes in README

### PR DESCRIPTION
**Pull request by gauravsinghaec**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
 # GoogleUdacity_JeevanRakht
 
-The code and development of the JeevanRakht's website is housed here.   
-For more details - [Refer to this paper.](https://paper.dropbox.com/doc/JeevanRakht-WebD-Project-ZroUbusvKbwRSRCDFHCOV)
+JeevanRakht is a Virtual Blood Bank web app, that aims at providing mechanism for people in need to directly talk to willing donors, and also find out blood availability in nearby blood banks.
+On this platform people willing to donate blood can sign up with details of their blood groups, location. Also there’ll be accounts of blood banks with details like number of packages of blood available and blood group, location of blood bank to filter out nearest blood banks.
+You can find the project's wireframe below which depicts the basic flow of our web application.
+
+## Classified Documents (Access restricted to Mods/Co-Mods only)
```
Comment (viditkothari) [inline README.md]: You can remove the restricted / private resources from this public doc :)

File: README.md [hunk 2]
```diff
@@ -25,22 +29,45 @@ https://jeevanrakht.herokuapp.com/
...
-* \models
-* \routes
-* \views
-* \controllers
-* app.js
+```
+Note : The folder structure may changes i.e we may include/exclude some folders/files as project progresses but the 
+overall sructure will remain as presented below:
+```
+### Backend Part (node.js, express.js with MongoDB)
```
Comment (viditkothari) [inline README.md]: Perfect!

File: README.md [hunk 3]
```diff
@@ -25,22 +29,45 @@ https://jeevanrakht.herokuapp.com/
...
+* \models       -- MongoDB collections schema
+* \routes       -- Apllication code for handling different UI routes e.g /login, /users etc
+* \views        -- Apllication related htmls files related to application routes
+* \controllers  -- Methods to interact with DB and reflects the data on views
+* \utils        -- Common Utility functions used in application
+* app.js        -- Application driver code
 
 ### Frontend Part
-* \public   -- html files
+* \public
```
Comment (viditkothari) [inline README.md]: Perfect!

File: README.md [hunk 4]
```diff
@@ -25,22 +29,45 @@ https://jeevanrakht.herokuapp.com/
...
+    - app.js    -- JS for our application
+    - main.js   -- JS for website
+  - images      -- Put your images here
+  - optimized_images  -- we will create optimized images using grunt & image magick to optimize site/app performance.
+  - scss        -- We are not using it as most people are not familier with scss
+
 * gulpfile.js -- This is a task runner to launch app and monitor for file(scss,css,js) changes and reloads the browser.
 
 ### Common to both Frontend and Backend
+* \views -- This will have htmls for our app (the htmls are with .hjs extension)
```
Comment (viditkothari) [inline README.md]: Perfect!

File: README.md (hunk 1 above)
Comment (viditkothari) [inline README.md]: I think,  you can remove the private section altogether and you will be good to go and merge.

**Comment by viditkothari** [comment]
@gauravsinghaec , excuse me for some extras here. I was trying to use various GitHub functionalities ;)

File: README.md (hunk 1 above)
Comment (gauravsinghaec) [reply README.md]: okay will remove it and merge.

**Comment by gauravsinghaec** [comment]
Updated as per review comment.

**Pull request merged**